pkg/components/util/helm.go

```diff
@@ -59,7 +59,11 @@ func RenderChart(helmChart *chart.Chart, name, namespace, values string) (map[st
 
 	// Include hooks
 	for _, m := range template.Hooks {
-		ret[m.Path] = m.Manifest
+		if val, ok := ret[m.Path]; ok {
+			ret[m.Path] = val + "\n---\n" + m.Manifest
+		} else {
+			ret[m.Path] = m.Manifest
+		}
```

Comment:
I think this code can be simplified to something like:
```suggestion
                if _, ok := ret[m.Path]; !ok {
                        ret[m.Path] = ""
                }
                
                ret[m.Path] += "---\n" + m.Manifest
```

Reply:
OK. I didn't read contrib guidlines. Shame on me....
I'll update my PR asap.

Reply:
Ping @jbanety, are you still interested in doing this work?

Reply:
Hi @invidian,
Sorry. Yes I am. Little busy at the moment.
I'll try to work on it this week